src/main/scripts/org/reaktivity/specification/nukleus/mqtt/streams/publish.fails/client.rpt

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
Why does publish fail in this case?
Also, if first publish for a given topic is rejected, then we wouldn't want to create an application stream at all, right?